tests/parens/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2927,10 +2974,12 @@ try {
 }
 
 @list($width, $height) = getimagesize($file);
-@(list($width, $height) = getimagesize($file));
+// Todo https://github.com/glayzzle/php-parser/issues/356
+// @(list($width, $height) = getimagesize($file));
 
 @$var += 10;
-@($var += 10);
+// Todo https://github.com/glayzzle/php-parser/issues/356
+// @($var += 10);
```

Comment:
Bug in parser

Reply:
Also parser has invalid ast before and now, so we can merge this commit and continue development, our plugin doesn't break output, only `AST_COMPARE` fails